windows/utils/win_enable_vss_log.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Enable VMware tools vss log in tools.conf
+- name: Set fact of VMware tools config file path
+  set_fact:
+    vmtools_conf_path: "C:\\ProgramData\\VMware\\VMware Tools\\tools.conf"
+    vmtools_conf_template: "C:\\ProgramData\\VMware\\VMware Tools\\tools.conf.example"
+
+# Check if VMware tools config file exists
+- include_tasks: win_check_file_exist.yml
+  vars:
+    win_check_file_exist_file: "{{ vmtools_conf_path }}"
+
+# Copy VMware tools config file from template file
+- include_tasks: win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "Copy-Item {{ vmtools_conf_template }} -Destination {{ vmtools_conf_path }}"
+  when: win_check_file_exist_result is defined and not win_check_file_exist_result 
+
+- name: Enable vss log in VMware tools config file
+  lineinfile:
+    path: "{{ vmtools_conf_path }}"
+    regexp: "{{ item.key }}"
+    line: "{{ item.value }}"
+  loop:
+    - {key: '^#log = true', value: 'log = true'}
+    - {key: '^#vmvss.level = debug', value: 'vmvss.level = debug'}
+    - {key: '^#vmvss.handler = vmx', value: 'vmvss.handler = vmx'}
+  delegate_to: "{{ vm_guest_ip }}"
+
+# Restart VMware tools service
+# - include_tasks: win_execute_cmd.yml
+#   vars:
+#     win_powershell_cmd: "Restart-Service -Force -Name VMTools"
```

Comment:
If this task is not required, I think we can remove it. The other looks good to me. Thanks!

Reply:
Sure, when testing done, I'll leave or remove the commented lines. Thanks.